### ISSUE TITLE
MPII val accuracy cannot reach 90.3

### ISSUE THREAD

**Issue by ChenyanWu** [leoxiaobin/deep-high-resolution-net.pytorch#163]
Recently I reproduced the experiments of HRnet and trained COCO and MPII dataset by myself. I find that the val accuracy of COCO dataset can exactly reach the accuracy in paper. But for MPII dataset my best result of val is 90.1, which is worse than the accuracy in paper 90.3. Is the parameter setting of this github repository exactly same with the parameter setting in paper?

**Comment by onepiece666** [comment]
my result is also 90.1.hava you changed the parameter and got the result of the paper

**Comment by zqylx** [comment]
My result is also 90.1, have you solved this problem?

**Comment by mobeixiaoxin** [comment]
@ChenyanWu ,my result is also 90.1, have you solved this problem?
Thanks!

**Comment by englishProgrammer** [comment]
you can get 90.03 while yot run `python tools/test.py \
    --cfg experiments/mpii/hrnet/w32_256x256_adam_lr1e-3.yaml \
    TEST.MODEL_FILE models/pytorch/pose_mpii/pose_hrnet_w32_256x256.pth`.

**Comment by ChenyanWu** [comment]
I finally get 90.3. I change the learning strategy. The total epoch is 140, and I change the learning rate at 90 and 120 epoch.

**Comment by zhanghao5201** [comment]
hao can i get 92.3?the implementation is only 90.3

**Comment by mobeixiaoxin** [comment]
> hao can i get 92.3?the implementation is only 90.3
hi，brother，how can you get the result of 90.3? I can get the result of 90.1, if convenience, can talk about this question by QQ?
my qq is 799428380! Thanks! 92.3 is the result on the Testdataset,you need to send you result to the http://human-pose.mpi-inf.mpg.de/

**Comment by zhanghao5201** [comment]
> I finally get 90.3. I change the learning strategy. The total epoch is 140, and I change the learning rate at 90 and 120 epoch.

How many GPUs and batchsize you use?

**Comment by ll8657** [comment]
> I finally get 90.3. I change the learning strategy. The total epoch is 140, and I change the learning rate at 90 and 120 epoch.

May I ask if you have solved this problem? I changed the learning strategy at 90 and 120epoch according to your method, but the accuracy of 140epoch in total is still less than 90.3 and only 89.3
